addons/account/models/partner.py

```diff
@@ -343,6 +343,8 @@ def _credit_debit_get(self):
         where_params = [tuple(self.ids)] + where_params
         if where_clause:
             where_clause = 'AND ' + where_clause
+        self.env['account.move.line'].flush_model()
+        self.env['account.account'].flush_model()
```

Comment:
flushing the whole tables are a bad practice. You should put your fields explicitely in `fnames`.
(the `JOIN` on `account_account` seems 1) useless and 2) should be a `JOIN` and not a `LEFT JOIN`. The account is required!)

Reply:
I flushed only the relevant fields and changed the LEFT JOIN to an (INNER) JOIN

I think we need join with account to determine the sign?
Or is there a better way?

Reply:
I removed the diff on the query as discussed in person